dsl/camel-jbang/camel-jbang-plugin-kubernetes/src/main/java/org/apache/camel/dsl/jbang/core/commands/kubernetes/KubernetesRun.java

```diff
@@ -499,6 +497,7 @@ private Integer deployProject(String workingDir) throws IOException, Interrupted
             args.add("k8s:deploy");
         }
 
+        printer().println(String.join(" ", args));
```

Comment:
The other commands does not print.

The jbang commands usually have a `--verbose` flag also. So maybe we should use that to print more details or not.

Reply:
ok, looking into it ...

Reply:
You can now explicitly turn on the --quiet mode for build/deploy